src/main/webapp/lectures/lectures.js

```diff
@@ -19,6 +19,8 @@ const NO_LANGUAGES_AVAILABLE_MESSAGE =
     'Sorry, there is no transcript available for this lecture.';
 const SELECT_LANGUAGE_OPTION_MESSAGE =
     'Select the language for the transcript.';
+const LANGUAGE_SELECTOR = 'language-input';
```

Comment:
this is a parameter, right? can u rename it PARAM

Reply:
Sort of, it's the name of selector input tag. It doesn't get added as a parameter to a URL though. I think it'll be more fitting to update it to `INPUT_LANGUAGE`